activity/rest/activity.go

```diff
@@ -119,7 +120,21 @@ func (a *RESTActivity) Eval(context activity.Context) (done bool, err error) {
 		req.Header.Set("Content-Type", contentType)
 	}
 
-	client := &http.Client{}
+	// Set the proxy server to use, if supplied
+	proxy := context.GetInput(ivProxy)
+	var client *http.Client
+	if proxy != nil && proxy.(string) != "" {
```

Comment:
You should use
v, ok := proxy.(string)

More here: https://golang.org/ref/spec#Type_assertions